spec/factories/placements/academic_years.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: academic_years
+#
+#  id         :uuid             not null, primary key
+#  ends_on    :date
+#  name       :string
+#  starts_on  :date
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+FactoryBot.define do
```

Comment:
I would move this to the `spec/factories/academic_years.rb`, similar to the `spec/factories/users.rb`
```
factory :placements_academic_year, class: "Placements::AcademicYear", parent: :academic_year
```